src/tools/pbs_upgrade_job.c

```diff
@@ -654,30 +812,33 @@ main(int argc, char *argv[])
 	/* Close the job file */
 	ret = close(fd);
 	if (ret < 0) {
-		fprintf(stderr, "Failed to close job file [%s]\n",
-			errno ? strerror(errno) : "No error");
+		fprintf(stderr, "Failed to close the job file [%s]\n",
+				errno ? strerror(errno) : "No error");
 		return 1;
 	}
 
 	/* Upgrade the task files */
 	dir = opendir(taskdir);
 	if (!dir) {
-		fprintf(stderr, "Failed open task directory [%s]\n",
-			errno ? strerror(errno) : "No error");
+		fprintf(stderr, "Failed to open the task directory [%s]\n",
+				errno ? strerror(errno) : "No error");
 		return 1;
 	}
 	while (errno = 0, (dirent = readdir(dir)) != NULL) {
 		if (errno != 0) {
 			fprintf(stderr, "Failed to read directory [%s]\n",
-				errno ? strerror(errno) : "No error");
+					errno ? strerror(errno) : "No error");
 			return 1;
 		}
 		if (dirent->d_name[0] == '.')
 			continue;
 		strcpy(task_start, dirent->d_name);
-		ret = upgrade_task_file(namebuf);
+		ret = upgrade_task_file(namebuf, job_version);
+		if (ret != 0) {
+			fprintf(stderr, "Upgrade task failed\n");
```

Comment:
As we are now throwing an error message in case of failure of an upgrade of task file, please mention the task file name for which the upgrade is failed.

Reply:
done!